src/main/java/com/google/api/gax/testing/LocalServiceHelper.java

```diff
@@ -258,5 +258,14 @@ public void sendPostRequest(String request) throws IOException {
     OutputStream out = con.getOutputStream();
     out.write("".getBytes());
     out.flush();
+
+    BufferedReader reader =
+        new BufferedReader(new InputStreamReader(con.getInputStream()));
+    String inputLine = "";
+    StringBuffer buffer = new StringBuffer();
+    while ((inputLine = reader.readLine()) != null) {
+        buffer.append(inputLine);
+    }
+    System.out.println("Local service response: " + buffer);
```

Comment:
I don't know that we want to automatically print the response. I think we should return the result through the return value, and the caller can use it or ignore it.